ert_shared/dark_storage/endpoints/ensembles.py

```diff
@@ -59,3 +59,10 @@ async def get_ensemble_userdata(
     ensemble_id: UUID,
 ) -> Mapping[str, Any]:
     raise NotImplementedError
+
+
+@router.get("/ensembles/{ensemble_id}/parameters", response_model=List[dict])
```

Comment:
if we change this to 
```
@router.get("/ensembles/{ensemble_id}/parameters", response_model=List[dict])
async def get_ensemble_parameters(
    *, res: LibresFacade = Depends(get_res), ensemble_id: UUID
) -> List[str]:
    name = get_name(type="ensemble", uuid=ensemble_id)
    return res.get_ensemble_parameters(name)
```
i think we should be able to get the correct parameters?

Reply:
So we replace ensemble_id with name in get_ensemble_parameters(). But its still not used for anything, perhaps this will change in the future ?

Reply:
in another comment i suggested to change the implementation in libres facade to this
```
def get_ensemble_parameters(self, ensemble_name: str) -> List[dict]:
        return [
            dict(
                name=key,
                values=parameter.values.tolist(),
            )
            for key, parameter in (
                (key, self.gather_gen_kw_data(ensemble_name, key))
                for key in self.all_data_type_keys()
                if self.is_gen_kw_key(key)
            )
        ]
``` 
instead of using `self.get_current_case_name()` use the actual case / ensemble name ( i THINK case is the same as ensemble name - but i might be confused and this is a wrong assumption )